support-frontend/assets/pages/contributions-landing/contributionsLanding.scss

```diff
@@ -837,6 +842,17 @@ form {
   width: 100%;
 }
 
+.form__submit--secure {
+  margin: -6px -10px 0;
```

Comment:
could we achieve similar top and bottoms for margins and padding using $gu-v-spacing / 2 instead? For code and vis consistency?

Reply:
Maybe - I couldn't get `$gu-v-spacing` to work with negative values but maybe we can pair on this?